bicep/files-to-load/create_ccw_template.py

```diff
@@ -0,0 +1,258 @@
+import argparse
```

Comment:
We should be just creating a parameters.json, not a cluster template.
All of the features that we are adding to the slurm template must go into cyclecloud-slurm (or they do not go in to either project)